apps/interpreter/src/parse-only.spec.ts

```diff
@@ -0,0 +1,50 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+import * as path from 'path';
+import * as process from 'process';
+
+import { RunOptions } from '@jvalue/jayvee-interpreter-lib';
+
+import { runAction } from './run-action';
+
+describe('Parse Only', () => {
+  const baseDir = path.resolve(__dirname, '../../../example/');
+
+  const defaultOptions: RunOptions = {
+    env: new Map<string, string>(),
+    debug: false,
+    debugGranularity: 'minimal',
+    debugTarget: undefined,
+    parseOnly: true,
+  };
+
+  beforeEach(() => {
+    jest.spyOn(process, 'exit').mockImplementation(() => {
+      throw new Error();
+    });
+  });
+
+  it('should exit with 0 on a valid option', async () => {
+    await expect(
+      runAction(path.resolve(baseDir, 'cars.jv'), {
+        ...defaultOptions,
```

Comment:
These tests would be better if you explicitly define the `parseOnly` flag here instead of taking it from `defaultOptions`. That makes it easier to understand what is actually being tested here.